public/static/partials/subscription_confirmation_toast.html

```diff
@@ -0,0 +1,3 @@
+<md-toast>
+  The subscription has been added. It can be modified from the (forthcoming) subscriptions page.
```

Comment:
In this case, I'd just make this say "The subscription has been added".

Also, is there an EVG ticket for the subscriptions page?